brambling/views/orders.py

```diff
@@ -151,7 +151,8 @@ class SurveyStep(OrderStep):
 
     @classmethod
     def include_in(cls, workflow):
-        return workflow.event.collect_survey_data
+        return (workflow.event.collect_survey_data or
+                workflow.event.forms.filter(form_type=CustomForm.ORDER).exists())
```

Comment:
There's a small chance that this should check for fields rather than a form, but... w/e. If we get there it'll be a pretty drastic overhaul of the custom form stuff anyway, so no use trying to predict what we'll want to do.